src/curl.c

```diff
@@ -177,8 +188,7 @@ Obj FuncCURL_REQUEST(Obj self,
 
 // Table of functions to export
 static StructGVarFunc GVarFuncs[] = {
-    GVAR_FUNC(CURL_REQUEST, 6,
-              "url, type, out_string, verifyCert, verbose, followRedirect"),
+    GVAR_FUNC(CURL_REQUEST, 1, "list"),
```

Comment:
As discussed on gather: I believe this would also work; internally GAP maps this to a vararg function, i.e., `FuncCURL_REQUEST` gets a single plist argument containing the seven original arguments -- so `FuncCURL_REQUEST` could stay unchanged; but in the GAP code you could again remove the `[...]` around the arguments to `CURL_REQUEST`
```suggestion
    GVAR_FUNC(CURL_REQUEST, 7,
               "url, type, out_string, verifyCert, verbose, followRedirect, failOnError"),
```